src/components/form/inputs/CreatableMultiSelectInput.tsx

```diff
@@ -58,7 +58,7 @@ export default function CreatableMultiSelectInput({
       >
         <R>
           <CreatableMultiSelect
-            selectStyle={{ border: 'none' }}
+            selectStyle={{ border: 'none', zIndex: '999' }}
```

Comment:
this is a brittle solution what happens if we have another modal or screen that needs to be on top?